Rakefile

```diff
@@ -12,6 +12,24 @@ end
 
 spec.urls.each { |url| url.chomp! }
 
+desc "task for fast iteration"
```

Comment:
Ignore this task, it's just to help me in development, i committed in case anyone else wants to work on this branch. I'll remove it before opening a PR upstream